virttest/qemu_storage.py

```diff
@@ -986,9 +986,12 @@ def convert(self, params, root_dir, cache_mode=None,
         convert_cmd = self.image_cmd + " " + \
             self._cmd_formatter.format(self.convert_cmd, **cmd_dict)
 
-        LOG.info("Convert image %s from %s to %s", self.image_filename,
-                 self.image_format, convert_image.image_format)
-        process.run(convert_cmd)
+        timeout = params.get_numeric("conversion_timeout", 12000)
```

Comment:
Do we consider to support the case of not setting timeout? I'm afraid the no timeout case won't be parsed properly by calling `Param.get_numeric`. Otherwise, as a workaround, users have to set a very large value to simulate that.

Reply:
Why is that? By default no timeout will be represented as no `conversion_timeout` key here which would result in `get_numeric` returning 12000, am I missing something? Or by "not parsed properly" do you mean "not handled properly" as in "forced to 12000 seconds"?

Reply:
I mean to exactly support setting no timeout rather than a certain time limit. In that case, no timeout would be logically treated as NaN, which should not be parsed by `get_numeric`.

Reply:
Then I propose (and will push) an option to do
```python
        timeout = convert_params.get_numeric("image_conversion_timeout", -1)
        timeout = None if timeout == -1 else timeout
```
even though I can think of no practical situation where infinite duration is reasonable as a use case. Of course, it is just as hard to guess large enough values for all sorts of QCOW2 conversions and sizes so I will simply push that change, treating -1 as no timeout.